mica/web/close_events.py

```diff
@@ -0,0 +1,24 @@
+from kadi import events
+from Chandra.Time import DateTime
+
+# 30 minute pad for these events
+interest_pad = 30 * 60;
+interest = [events.normal_suns,
+            events.safe_suns,
+            events.scs107s,
+            events.caps]
+
+def close_events(start, stop):
```

Comment:
My thought here was that most of the time if I'm interested in "what happened around this time" it would be handy to know if any already-known interesting events occurred close to the time of interest without having to do a separate search in another page.